rancher/shell@3.0.10 causes build-extension-catalog failure

<!--------- For bugs and general issues --------->
**Setup**
- Rancher version: 2.13.3
- Rancher UI Extensions: Rancher Supportability Review 3.0.3
- Browser type & version:

**Describe the bug**
<!--A clear and concise description of what the bug is.-->

rancher/shell@3.0.10 requires node.js 24 but `build-extension-catalog.yml` uses node.js 20 [1]. So following error occurs.
```
yarn install v1.22.22
[1/5] Validating package.json...
error supportability-review-app@3.0.3: The engine "node" is incompatible with this module. Expected version ">=24". Got "20.20.1"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Error: Process completed with exit code 1.
```

`build-extension-charts.yml` works fine because it uses `.nvmrc` [2].

[1] https://github.com/rancher/dashboard/blob/master/.github/workflows/build-extension-catalog.yml#L77
[2] https://github.com/rancher/dashboard/blob/master/.github/workflows/build-extension-charts.yml#L78

**To Reproduce**
<!--Steps to reproduce the behavior-->
Run release for Rancher Supportability Review 3.0.3

**Result**
Release failed.

**Expected Result**
<!--A clear and concise description of what you expected to happen.-->

**Screenshots**
<!-- If applicable, add screenshots to help explain your problem and/or errors in the browser's dev console -->

**Additional context**
<!--Add any other context about the problem here. -->

--- a/.github/workflows/build-extension-catalog.yml
+++ b/.github/workflows/build-extension-catalog.yml
@@ -74,7 +74,7 @@ jobs:
       - name: Setup Nodejs with yarn caching
         uses: actions/setup-node@v6
         with:
-          node-version: '20'
+          node-version-file: ".nvmrc"
           cache: yarn
 
       - name: Install dependencies

